packages/core/lib/testing/Deployed.js

```diff
@@ -37,18 +36,20 @@ var Deployed = {
 
     source += "}";
 
-    if (version) {
-      if (version === "native") version = new Native().load().version();
-      const v = semver.coerce(version);
-      if (semver.lt(v, "0.5.0")) source = source.replace(/ payable/gm, "");
-      source = source.replace(/0\.5\.0/gm, v);
+    version = Version.resolveVersionString(version);
+    if (!Version.versionIsAtLeast(version, "0.5.0")) {
+      //remove "payable"s if we're before 0.5.0
+      source = source.replace(/ payable/gm, "");
```

Comment:
I know you didn't write this code but is it potentially bad to just replace this? Can you use `payable` as a variable name? Do we need to make sure that it has a space at each end like `/ payable /gm`?

Reply:
I mean, `address payable varName;` would be a syntax error before 0.5.0, right?  Dunno that this search-and-replace is the best way to handle it, but I don't think we can just leave it alone?

Reply:
Seems like we should replace `address payable` with `address`, no?

Reply:
Hm, yeah.  I guess I'll change that.  That's an easy change to make without having to figure out the actual proper way to go about things.